### DISCUSSION TITLE
iconAtlas using SVG

### DISCUSSION THREAD

**Discussion by txtl** [visgl/deck.gl#8136] [General]
In this example: https://deck.gl/docs/developer-guide/loading-data#customize-data-loading-behavior, there is mentioning of using SVG in iconAtlas. How does iconMapping look like? using prepacked PNG, iconMapping looks like this: 

const ICON_MAPPING = {
  marker: {x: 0, y: 0, width: 128, height: 128, mask: true}
};

for SVG, how to specify the x and y? 

Thanks for your inputs!

**Comment by Pessimistress** [comment]
Even if you use SVG, it's still loaded as a raster texture. You need to either specify width/height in the SVG or in loadOptions: #5158
